### PR TITLE
Add sonarqube healthcheck example

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sonarqube/docker-healthcheck [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# A Sonarqube container is considered healthy if the status is UP, DB_MIGRATION_NEEDED or DB_MIGRATION_RUNNING
+# status about migration are added to prevent the node to be kill while sonarqube is updating himself.
+
+if curl -Ss http://localhost:9000/api/system/status | grep -q -e '"status":"UP"' -e '"status":"DB_MIGRATION_NEEDED"' -e '"status":"DB_MIGRATION_RUNNING"'; then
```
Comment (tianon) [inline sonarqube/docker-healthcheck]: Shouldn't this be attempting to connect to the external interface instead to better verify the container's purported external health?

(As in https://github.com/docker-library/healthcheck/blob/436d3437d3b75f41d3518720c077df3e8d3570ce/cassandra/docker-healthcheck#L4 andd other examples here?)

File: sonarqube/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM sonarqube
+
+COPY --chown=sonarqube:sonarqube docker-healthcheck /usr/local/bin/
```
Comment (tianon) [inline sonarqube/Dockerfile]: What's the purpose of the `--chown=` here?

File: sonarqube/docker-healthcheck (hunk 1 above)
Comment (ghost) [reply sonarqube/docker-healthcheck]: Yes it is ! thanks, i'm updating the PR to do so.

File: sonarqube/Dockerfile (hunk 2 above)
Comment (ghost) [reply sonarqube/Dockerfile]: a mistake :) i'm removing it from the PR. nice catch

**Comment by ghost** [comment]
Thanks for your review, i fixed the 2 points in a 2nd commit.

**Review by tianon — APPROVED** [review]

**Pull request merged**